### PR TITLE
[14.0][ADD] partner_uom_mapping

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: partner_uom_mapping/views/partner_uom.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[]</field>
+        <field name="context">{'search_default_group_by_partner': 1}</field>
+    </record>
+
+    <record model="ir.ui.menu" id="partner_uom_menu">
+        <field name="name">Partner Uom</field>
+        <field name="parent_id" ref="contacts.res_partner_menu_config" />
+        <field name="action" ref="partner_uom_act_window" />
+        <field name="groups_id" eval="[(4, ref('group_partner_uom'))]" />
```
Comment (remytms) [inline partner_uom_mapping/views/partner_uom.xml:68]: Isn't it a bit confusing that, giving the `group_partner_uom` let you access to the partner.uom in the UI, but cannot modify it if the user is not part of the `base.group_partner_manager` ?

I may be wrong, but, I was expecting that `group_partner_uom` allow `base.group_user` to see the menu and just read the `partner.uom`, and that `base.group_partner_manager` can always see this menu to read and write the `partner.uom`.

File: partner_uom_mapping/demo/partner_uom.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+
+    <!-- TODO
+    <record model="partner.uom" id="partner_uom_demo_1">
+       <field name="name">...</field>
+    </record>
+    -->
+
+</odoo>
```
Comment (remytms) [inline partner_uom_mapping/demo/partner_uom.xml]: I think that this file can be deleted or completed with demo data. :)

File: partner_uom_mapping/views/partner_uom.xml (hunk 1 above)
Comment (rousseldenis) [reply partner_uom_mapping/views/partner_uom.xml:68]: README adapted.

**Review by remytms — APPROVED** [review]
I did a code review. Some comments where taken into account.

**Review by marielejeune — APPROVED** [review]
LGTM (code review)

**Pull request closed without merging**